afu_isboden_pub/afu_isboden_pub_bodeneinheit_gesamt.sql

```diff
@@ -0,0 +1,273 @@
+SELECT
+    pk_ogc_fid AS t_id,
+    gemnr,
+    objnr,
+    wasserhhgr,
+    wasserhhgr_beschreibung,
+    wasserhhgr_qgis_txt,
+    bodentyp,
+    bodentyp_beschreibung,
+    gelform,
+    CASE
+        WHEN gelform = 'a' OR gelform = 'b' OR gelform = 'c' OR gelform = 'd' OR gelform = 'e'
```

Comment:
Falls das geht, nutze hier besser `WHEN gelform IN ('a', 'b', 'c', 'd', 'e')`, da besser lesbar.

Gilt analog auch auf diversen anderen Zeilen.

Oder besser gleich nochmal eleganter:
```
CASE gelform
    WHEN 'a', 'b', 'c', 'd', 'e'
        THEN '0-10%: Keine Einschränkung'
    WHEN ...usw.
```
Doku s. https://www.postgresql.org/docs/10/static/plpgsql-control-structures.html#PLPGSQL-CONDITIONALS unter _42.6.2.4. Simple CASE_